### PR TITLE
UPS-3487 - add button exchange

### PR DESCRIPTION
**Pull request by simon-debruijn**
ticket: https://jira.uitdatabank.be/browse/UPS-3487

### CODE REVIEW

File: src/uitpas/search/detail/UiTPasAdvantageDescription.js [hunk 1]
```diff
@@ -124,6 +135,32 @@ export default class UiTPasAdvantageDescription extends React.Component {
...
+        if (this.isOnlineExchangable()) {
+            return <div className="online-exchangable">online om te ruilen</div>
+        }
+
+        return null;
+    }
+
+    onlineExchangeUrl() {
+        console.log('env', process.env.REACT_APP_UITPAS_URL);
+        return `${process.env.REACT_APP_UITPAS_URL}/omruilen/${this.props.advantage.id}`;
```
Comment (bertramakers) [inline src/uitpas/search/detail/UiTPasAdvantageDescription.js:153]: Can't you use the existing `PUBLIC_URL` here?

File: src/uitpas/search/detail/UiTPasAdvantageDescription.js [hunk 2]
```diff
@@ -124,6 +135,32 @@ export default class UiTPasAdvantageDescription extends React.Component {
...
+    renderIsOnlineExchangable() {
+        if (this.isOnlineExchangable()) {
+            return <div className="online-exchangable">online om te ruilen</div>
+        }
+
+        return null;
+    }
+
+    onlineExchangeUrl() {
+        console.log('env', process.env.REACT_APP_UITPAS_URL);
```
Comment (bertramakers) [inline src/uitpas/search/detail/UiTPasAdvantageDescription.js:152]: Remove the `console.log` 😉

File: src/uitpas/search/detail/UiTPasAdvantageDescription.js [hunk 3]
```diff
@@ -4,9 +4,14 @@ import { map } from 'lodash';
...
 
 import { UiTImage } from '../component/UiTImage';
-import { UiTPasCounter } from './UiTPasCounter';
 import { LastChanceLabel } from '../component/LastChanceLabel';
 import { joinNicely } from '../helper/UiTPasArrayUtils';
+import UiTPasSearchConfig from '../UiTPasSearchConfig';
+
+const CashInType = {
+    ONLINE: 'ONLINE',
+    FYSICAL: 'FYSICAL',
```
Comment (bertramakers) [inline src/uitpas/search/detail/UiTPasAdvantageDescription.js:13]: I guess there is no way to use the correct spelling (`physical`)?

File: src/sass/_button.scss [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+  font-weight: 600;
+  width: 100%;
+  padding: 1.5rem 1rem;
+  margin-bottom: 1rem;
+  border: none;
+  color: white;
+  background-color: blue;
+  cursor: pointer;
+  background-color: #1aa5a8;
+}
```
Comment (bertramakers) [inline src/sass/_button.scss:13]: Nit: Should probably have a trailing newline

File: src/uitpas/search/detail/UiTPasAdvantageDescription.js (hunk 1 above)
Comment (simon-debruijn) [reply src/uitpas/search/detail/UiTPasAdvantageDescription.js]: No, it's something else

File: src/uitpas/search/detail/UiTPasAdvantageDescription.js (hunk 3 above)
Comment (simon-debruijn) [reply src/uitpas/search/detail/UiTPasAdvantageDescription.js]: I removed it since it wasn't used

File: src/sass/_button.scss (hunk 4 above)
Comment (simon-debruijn) [reply src/sass/_button.scss]: I introduced prettier

File: src/sass/_button.scss [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+  text-align: center;
+  font-weight: 600;
+  width: 100%;
+  padding: 1.5rem 1rem;
+  margin-bottom: 1rem;
+  border: none;
+  color: white;
+  background-color: blue;
+  cursor: pointer;
+  background-color: #1aa5a8;
```
Comment (brampauwelyn) [inline src/sass/_button.scss:12]: Maybe make a sass variable?

File: src/uitpas/search/hits/UiTPasSearchHits.js [hunk 6]
```diff
@@ -1,103 +1,131 @@
...
-    ResetFilters,
-    SortingSelector,
-    PageSizeSelector,
-    Select,
-} from 'searchkit';
+  SearchkitComponent,
+  Hits,
+  NoHits,
+  InitialLoader,
+  // Pagination,
```
Comment (brampauwelyn) [inline src/uitpas/search/hits/UiTPasSearchHits.js:7]: Can we remove this one?

File: src/uitpas/search/detail/UiTPasAdvantageDescription.js (hunk 1 above)
Comment (bertramakers) [reply src/uitpas/search/detail/UiTPasAdvantageDescription.js]: Please clarify next time what is then next time like @brampauwelyn did here https://github.com/cultuurnet/uitpas-voordelen-search/pull/31#discussion_r664423832

**Comment by bertramakers** [comment]
Was this merged without an approval? We should set up a branch protection rule on this repository then.

**Comment by brampauwelyn** [comment]
> Was this merged without an approval? We should set up a branch protection rule on this repository then.
> Edit: There is already a review requirement check for PRs that go into `master`. How did you merge this without an approval from @brampauwelyn or me?

I think it was merged before the requirement was enabled. But we paired on it so the changes are all good to go :)

**Comment by simon-debruijn** [comment]
> > Was this merged without an approval? We should set up a branch protection rule on this repository then.
> > Edit: There is already a review requirement check for PRs that go into `master`. How did you merge this without an approval from @brampauwelyn or me?
> 
> I think it was merged before the requirement was enabled. But we paired on it so the changes are all good to go :)

I added the rules for master branch very recently, so ⬆️ is very likely

**Comment by bertramakers** [comment]
👍

**Pull request merged**